test/integration/render/try.spec.tsx

```diff
@@ -1,6 +1,7 @@
 import driver from "@plusnew/driver-dom/src/driver";
 import "@plusnew/driver-dom/src/jsx";
 import plusnew, { Async, component, context, store, Try } from "../../../index";
+import ComponentInstance from "../../../src/instances/types/Component/Instance";
```

Comment:
better to use an absolute path here

Reply:
Sadly that's not possible. Libraries will have the problem that the parent project will try to resolve it from their own root. Therefore all paths need to be relative.